Cartfile

```diff
@@ -1,2 +1,2 @@
 binary "https://raw.githubusercontent.com/AppsFlyerSDK/AppsFlyerFramework/master/Carthage.json" ~> 4.8.0
-github "mparticle/mparticle-apple-sdk" ~> 7.4.0
+github "mparticle/mparticle-apple-sdk" ~> 7.5.0
```

Comment:
Should this be 7.5.1?

Reply:
It isn't necessary.
Appsflyer kit 7.5.1 depends on mParticle-Apple-SDK versions that are compatible with 7.5.0.
All versions that are lower than 7.6.0 is considered compatible with 7.5.0 according to Semantic Versioning.
As 7.5.x versions become available, carthage update command will update which version of mParticle-Apple-SDK would be used.

Cartfile and .podspec changes are pulled from the origin and shouldn't require any changes on our end.